### PR TITLE
Save input into local storage

### PR DESCRIPTION
**Pull request by Drumor**

### CODE REVIEW

File: inginious/frontend/templates/task.html [hunk 1]
```diff
@@ -436,6 +439,41 @@ <h4 class="modal-title" id="socialModalLabel">$:_("Share my result on: ")</h4>
 
 </script>
 
+<script type="text/javascript">
+    $$("#load_loc").on('click', function(e){
+        alert("Loaded from browser");
```
Comment (anthonygego) [inline inginious/frontend/templates/task.html]: Is this supposed to actually display the message box ?

File: inginious/frontend/templates/task.html [hunk 2]
```diff
@@ -436,6 +439,41 @@ <h4 class="modal-title" id="socialModalLabel">$:_("Share my result on: ")</h4>
...
+            {
+               editor.setValue(localStorage["$course.get_id()"+"/"+name]);
+            });
+        $$("input:text").each(function(){
+            let name = "$course.get_id()"+"/"+($$(this).attr("name"));
+            $$(this).val(localStorage[name]);
+        });
+    });
+    $$("#save_loc").on('click', function(e){
+        alert("Saved to browser");
```
Comment (anthonygego) [inline inginious/frontend/templates/task.html]: Same comment here

File: inginious/frontend/templates/task.html (hunk 1 above)
Comment (Drumor) [reply inginious/frontend/templates/task.html]: Removed... was for test purposes

File: inginious/frontend/templates/task.html (hunk 2 above)
Comment (Drumor) [reply inginious/frontend/templates/task.html]: Removed... was for test purposes

**Comment by Drumor** [comment]
I finished this one (from my point of view ofc :D )

**Comment by Drumor** [comment]
Hello,

I added the taskid in the key.

**Comment by anthonygego** [comment]
I'll make further tests, but LGTM

**Pull request merged**